devtools/client/README.md

```diff
@@ -1,3 +1,62 @@
-# devtools-client
+# @player-tools/devtools-client
```

Comment:
Does the `js-pipeline` macro pull in the README automatically? We should add it if it doesn't yet

Reply:
It doesn't, the glob only gets [src](https://github.com/player-ui/rules_player/blob/e71339c96deeec1f4976fbc04815143ee23d8541/javascript/private/js_pipeline.bzl#L61). I'll add it to the #100.